Content.Server/Chemistry/ReagentEffects/ResetNarcolepsy.cs

```diff
@@ -0,0 +1,27 @@
+using Content.Server.Traits.Assorted;
+using Content.Shared.Chemistry.Reagent;
+using JetBrains.Annotations;
+
+namespace Content.Server.Chemistry.ReagentEffects
```

Comment:
```suggestion
namespace Content.Server.Chemistry.ReagentEffects;
```
File-scoped namespaces are preferred for new .cs files.